Pricing docs (#901)
sourcegraph/docs: docs/enterprise-starter.mdx

# Enterprise Starter

> <p className="subtitle">Learn about the Enterprise Starter plan tailored for teams and individuals.</p>

Reviewer: Is this the marketing we are doing for it? @kukicado  
I thought by calling it "enterprise starter" we are less focused on individuals?

Reviewer 2: I'd maybe swap "teams and individuals" to "individuals and teams"

The enterprise starter plan is meant for users that want private code indexing, private code search, and that entails so that includes individual devs that want to better leverage the SG platform.

> <Callout type="info">Enterprise Starter is available for Sourcegraph versions 6.0+ and supports VS Code, JetBrains, Cody Web.</Callout>

Reviewer: Since Enterprise Starter is not an on-prem feature, does it matter the version?

Reviewer 2: Yeah I would be ok removing this callout.

> The Enterprise Starter plan offers a multi-tenant Sourcegraph designed for small teams and individuals that require the same core features of a traditional Sourcegraph instance but with a simplified management experience. This plan provides a fully managed version of Sourcegraph (AI + code search + omnibox, with privately indexed code) through a self-serve flow.

Reviewer 2: ```suggestion
The Enterprise Starter plan offers a multi-tenant Sourcegraph instance designed for individuals and teams offering the core features of a traditional Sourcegraph instance but with a simplified management experience. This plan provides a fully managed version of Sourcegraph (AI + code search + omnibox, with privately indexed code) through a self-serve flow.
```

## Team seats

The Enterprise Starter plan is priced at **$19 per month per seat**. You can add or remove team members at any time. Existing Cody Pro users can also sign up for the Enterprise Starter by paying $19 per seat. However, their Cody Pro subscription will neither be upgraded nor canceled. Instead, they will have two live subscriptions.

## Enterprise Starter team roles

The Enterprise Starter plan includes the following team-level roles:

> - **Admin**: Has full access to the workspace, including the ability to manage users, billing, and settings

Reviewer: Should we explicitly say admin can manage the repos?

Author: Yes definitely. 👍

> - **Member**: Can access repositories in the workspace and can experience the supported features

Reviewer 2: ```suggestion
- **Member**: Can access repositories in the workspace and use the supported features
```

## Billing

> Workspaces on the Enterprise Starter plan are billed monthly based on the number of team seats purchased. In case of overdue or failing payment, there is a grace period during which the workspace admins will receive a daily notification to complete the transaction.

Reviewer 2: I would add what happens if payment is not made (i.e workspace is deleted after x days and data is unrecoverable)

## Features supported

The Enterprise Starter plan supports a variety of AI and search-based features like:

> |             **AI features**              |        **Code Search**         |     **Deployment**     |                      **Management**                       |        **Support**        |

Reviewer: The deployment column doesn't make sense to me, not sure we need it

Author: I am waiting on the finalized message from Kelvin. Once that's done I'll have a final copy for it.

| ---------------------------------------- | ------------------------------ | ---------------------- | --------------------------------------------------------- | ------------------------- |

> | Unlimited autocomplete and chat messages | Indexed Code Search            | Multi-tenant cloud     | Simplified admin experience with UI-based repo-management | Support with limited SLAs |

Reviewer: @eseliger @dadlerj @kukicado  Are we making unlimited LLM calls a selling point for Enterprise Starter?

Author: I got this from the new Pricing page.

Reviewer 2: I'll defer to Kelvin on this one as he'll have the most up to date messaging. From my understanding, we are going to keep the "unlimited" messaging like we have in Pro for now for enterprise starter.

| Powerful LLM models for chat             | Indexed Symbol Search          | Private workspace      | User management                                           | -                         |
| Omnibox intent detection                 | Searched based code-navigation | Privately indexed code | GitHub code host integration                              | -                         |
| Cody integration                         | -                              | -                      | -                                                         | -                         |

> <Callout type="note">We are actively working on bringing more features to Enterprise Starter. Stay tuned for updates!</Callout>

Reviewer 2: I would remove this callout. I don't think it makes sense for the docs experience.

## Limits

Sourcegraph Enterprise Starter offers the following limits:

- Max 50 users per workspace

> - Max 100 repos

Reviewer 2: ```suggestion
- Max 100 repos per workspace
```

> - 5 GB of storage

Reviewer: Maybe instead say "Starts with 5 gb storage"

Author: Is there an upper limit?

Reviewer: Each workspace starts with 5 gigs of storage and then gets 1 additional gig per user added up to 10 gigs. 

We will likely revisit these limits in the future but this is our starting point

Reviewer 2: @mmanela just wanted to confirm that it is an additional 1GB per user, I heard elsewhere that it's an extra 500mb per user, up to the 10GB upper limit.

Reviewer: It is additional 1 gig per user up to 10gigs

- 1 GB storage per seat added
- 10 GB max total storage

## Workspace settings

After creating a new workspace, you can switch views between your personal and workspace accounts. Once you are in your workspace, you can configure the **Workspace settings**. These include options for:

- **General Settings**: Helps you configure how your workspace is described or accessed with options like workspace name, URL, and deleting the workspace
- **Users**: Manage permissions, assign seats, or invite new users
- **Billing**: Manage your monthly billing cycle for all your purchased seats

> - **Repository Management**: A complete track and log of your connected repos with the storage used

Reviewer: Instead maybe

`Add, remove and view the status of your connect repositories`
